README.md

```diff
@@ -1,2 +1,60 @@
-# ansible-falcon
-Ansible role to download and install CrowdStrike Falcon sensor
+Role Name
+=========
+The purpose of this role is to download and configure the Falcon to install it on our EC2 instances. The sensor must be downloaded from Crowdstrike Cloud using MFA. 
+
+In the first step, we will connect to the mentionced cloud environment to obtain a `Bearer token` which will be used to download the `pkg`. The distribution used is compatible with Ubuntu 16/18 and 20.
+
+After downloading and installing the `pkg` - the `falcon-sensor` must be configured. According to the official documentation both the Customer ID (CID) and `provisioning-token` must be provided.
+
+
+Requirements
+------------
+
+![Ansible](https://img.shields.io/badge/ansible-2.8.2-green)
+
+Role Variables and Secrets
+--------------
+
+Supported variables:
+
+* `falcon_cloud` - CrowdStrike API URL for downloading the Falcon sensor.
+* `falcon_instaler_id` - Installer ID: default version to be installed
+* `falcon_install_tmp_dir` - Where should the sensor file be downloaded to on Linux
+
+Supported secrets:
+
+This information can be found on under Falcon UI and won't be changed in the near future:
+
+* `falcon_client_id` - CrowdStrike OAUTH Client ID
+* `falcon_client_secret` - CrowdStrike OAUTH Client Secret
+* `falcon_cid` - CrowdStrike Customer ID
```

Comment:
I think this should have a clearer name:

```suggestion
* `falcon_customer_id` - CrowdStrike Customer ID
```

Reply:
It's true, changing since it can be misleading